webapi/src/main/twirl/queries/sparql/v2/changeLinkTarget.scala.txt

```diff
@@ -127,7 +128,7 @@ INSERT {
             knora-base:valueHasRefCount @linkUpdateForNewLink.newReferenceCount ;
             knora-base:valueHasOrder ?order ;
             knora-base:isDeleted false ;
-            knora-base:valueHasUUID "@{stringFormatter.base64EncodeUuid(UUID.randomUUID)}" ;
+            knora-base:valueHasUUID "@{stringFormatter.base64EncodeUuid(newLinkValueUUID)}" ;
```

Comment:
When the target of a link is changed, then the UUID is changed to? Is this because it has to be considered a new value (not a different version of the same value)?

Reply:
That's right. This was a design decision that Lukas and I made a long time ago. We thought that if you change the `rdf:object` of a reification, it wouldn't make sense to consider it a new version of the same reification, because it no longer reifies the same triple. So when you change a link's target, the old `LinkValue` is deleted, and a new one is created.

Reply:
I figure when you just change the comment, the uuid stays the same, right?

Reply:
Yes, if you just change the comment or any other metadata.